src/modules/auth/refreshToken.repository.ts

```diff
@@ -1,32 +1,32 @@
 import { Injectable } from '@nestjs/common';
 import { RefreshToken } from './refreshToken.model';
-import redis, { RedisClient } from 'redis';
+import { createClient as redisCreateClient, RedisClientType } from 'redis';
```

Comment:
Looks like more readable would be 
```suggestion
import { createClient as createRedisClient, RedisClientType } from 'redis';
```